core/templates/services/svg-sanitizer.service.ts

```diff
@@ -71,21 +71,33 @@ export class SvgSanitizerService {
       // we need to replace these decimals with a letter so that it's easier
       // to process and validate the filenames.
       if (node.tagName.toLowerCase() === 'svg') {
-        dimensions.height = (
-          (node.getAttribute('height').match(/\d+\.*\d*/g)[0]).replace(
-            '.', 'd'));
-        dimensions.width = (
-          (node.getAttribute('width').match(/\d+\.*\d*/g)[0]).replace(
-            '.', 'd'));
+        let attrHeight = node.getAttribute('height');
+        if (attrHeight) {
+          let height = attrHeight.match(/\d+\.*\d*/g);
+          if (height) {
```

Comment:
what happens to the properties of `dimension` if `height` and `width` do not exist?

Reply:
if height and width do not exist then default values i.e. empty strings will be used. `dimensions` have been initialized already.